components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -80,15 +80,27 @@ Views.loginOpened...
 There are different metrics to choose from, depending on what you want to achieve:
 
 * [Events](#Events): Records events e.g. individual occurences of user actions, say every time a view was open and from where.
```

Comment:
I'm not sure where it should fit, but I now realize that we really want to express our opinion with respect to metric names: maybe we should explain why `events.button_event` isn't a good naming scheme (e.g. event is redundant as a category name and button_event doesn't really explain what the event is about). Any chance you can tackle this here on file a follow-up for it?

Reply:
That sounds orthogonal to the PR here.
Can you file a follow-up on what your intent there is @Dexterp37  ?

Reply:
Filed this. https://bugzilla.mozilla.org/show_bug.cgi?id=1557700